packages/manager/apps/web/client/app/css/source.less

```diff
@@ -47,4 +47,30 @@
       box-shadow: none;
     }
   }
+
+  .website-coach-visualizer {
```

Comment:
You should use bootstrap utilities as much as possible.
Also this should be in a `webcoach.scss` imported in your module not in the global style of the app

Reply:
Bootstrap has no use here. I'll move that in the webcoach folder

Reply:
But other question, when has it been decided to use SCSS instead of LESS ?